### PR TITLE
Lookup collapse tokens (>5 and show more enabled)

### PR DESCRIPTION
**Pull request by Blackbaud-TrevorBurch**

### CODE REVIEW

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-lookup?_sp.spa=pr-lookup-collapse-tokens](https://developer.blackbaud.com/skyux-lookup?_sp.spa=pr-lookup-collapse-tokens)

File: src/app/public/modules/lookup/lookup.component.ts [hunk 1]
```diff
@@ -197,17 +202,34 @@ export class SkyLookupComponent
...
+      )
+        .pipe(take(1))
+        .subscribe((label) => {
+          this._tokens = [{
+            value: { [this.descriptorProperty]: label }
+          }];
+          this.changeDetector.markForCheck();
+        });
+    } else {
+      this._tokens = value;
```
Comment (Blackbaud-AlexKingman) [inline src/app/public/modules/lookup/lookup.component.ts:219]: Should we follow this branch with a `this.changeDetector.markForCheck();` as well?

**Review by Blackbaud-AlexKingman — COMMENTED** [review]
Works great. Just one minor comment.

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-lookup?_sp.spa=pr-lookup-collapse-tokens](https://developer.blackbaud.com/skyux-lookup?_sp.spa=pr-lookup-collapse-tokens)

**Review by Blackbaud-AlexKingman — APPROVED** [review]

**Pull request merged**